src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -538,3 +540,179 @@ export const sampleData: RawSpeciesStats = {
+    assembly_name: 'GRCh38.p13',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Ensembl',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'full genebuild',
+    assembly_date: '2019-02-01',
+    notes: [
+      {
+        heading: 'Pseudoautosomal regions',
+        body:
+          'The Y chromosome in this assembly contains two psuedoautosomal regions (PARs) that were taken from the corresponding X chromosoles and are exact duplicates:\nchrX:10,000-2,781,479 = chrY:10,000-2,781,479\nchrX:155,701,382-156,030,895 = chrY:56,887,02-57,217,415'
+      }
+    ]
+  },
+  homo_sapiens_GCA_000001405_14: {
+    id: 'GCA_000001405.14',
+    taxonomy_id: '9606',
+    strain: null,
+    database_version: '101.37',
+    common_name: 'Human',
+    scientific_name: 'Homo sapiens',
+    gencode_version: 'GENCODE 19',
+    assembly_name: 'GRCh37.p13',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Ensembl',
+      url: ''
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'full genebuild',
+    assembly_date: 'June 2013',
+    notes: []
+  },
+  escherichia_coli_str_k_12_substr_mg1655_GCA_000005845_2: {
+    id: 'GCA_000005845.2',
+    taxonomy_id: '511145',
+    strain: null,
+    database_version: '101.1',
+    common_name: 'Escherichia coli str. K-12 substr. MG1655',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'ASM584v2',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'European Nucleotide Archive',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Generated from ENA annotation',
+    assembly_date: 'Sep 2013',
+    notes: []
+  },
+  caenorhabditis_elegans_GCA_000002985_3: {
+    id: 'GCA_000002985.3',
+    taxonomy_id: '6239',
+    strain: {
+      type: 'Strain',
+      value: 'N2'
+    },
+    database_version: '101.269',
+    common_name: 'Caenorhabditis elegans',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'WBcel235',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'WormBase',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Import',
+    assembly_date: 'Feb 2013',
+    notes: []
+  },
+  plasmodium_falciparum_GCA_000002765_2: {
+    id: 'GCA_000002765.2',
+    taxonomy_id: '36329',
+    strain: null,
+    database_version: '101.1',
+    common_name: 'Plasmodium falciparum 3D7',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'ASM276v2',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Naval Medical Research Institute',
```

Comment:
In the db, this is assembly provider, not annotation provider. Curiously, there is no record of an annotation provider for Plasmodium falciparum:

![image](https://user-images.githubusercontent.com/6834224/97183140-4e877c00-1795-11eb-8997-932076a2ac80.png)

Reply:
That's done